src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,21 @@
+package duke.task;
+
+/**
+ * Represents a task that is ToDo in the Richie application
+ */
+
+public class Todo extends Task {
```

Comment:
Consider using pascal case naming convention, from `Todo` to `ToDo` since "to-do" might not be considered as a single word (i.e., "todo").